src/App.tsx

```diff
@@ -1,44 +1,28 @@
 import React from 'react';
 import './App.css';
 import { H1 } from './styles/typography/h1';
-import { H2 } from './styles/typography/h2';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faHeart } from '@fortawesome/free-solid-svg-icons';
+import { Form } from './components/forms/form';
+import { Separator } from './components/separator/separator';
 import { Button } from './components/buttons/button';
+import PropertyInfo from './components/property-info/propertyInfo';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCar } from '@fortawesome/free-solid-svg-icons';
+// import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
 
 function App() {
   return (
     <div className="App">
       <H1> Styleguide </H1>
-      <H2> Botões </H2>
-      <Button type="primary" icon={<FontAwesomeIcon icon={faHeart} />}>
-        primary button
-      </Button>
-      <div className={'spacing'}></div>
-      <Button type="secondary">secondary button</Button>
-      <div className={'spacing'}></div>
-      <Button type="cta">Ctabutton</Button>
-      <div className={'spacing'}></div>
-      <Button type="primary" icon={<FontAwesomeIcon icon={faHeart} />} compact>
-        primary button
-      </Button>
-      <div className={'spacing'}></div>
-      <Button type="secondary" compact>
-        secondary button
-      </Button>
-      <div className={'spacing'}></div>
-      <Button type="cta" compact>
-        Ctabutton
-      </Button>
-      <div className={'spacing'}></div>
-      <Button type="primary" disable>
-        primary button
-      </Button>
-      <div className={'spacing'}></div>
-      <Button type="primary" disable compact>
-        primary button
-      </Button>
-      <div className={'spacing'}></div>
+      <Form title="Nome"></Form>
+      <Separator className="spacing"></Separator>
```

Comment:
Essa classe `spacing` não poderia ficar dentro do componente Separator? Assim vc n precisa repeti-la toda a vez que for usar um separador.

Outro ponto é que existe um atalho pra fechar tags no jsx/tsx, quando não tem children:

```tsx
      <Separator />
```